v2/tools/generator/internal/armconversion/shared.go

```diff
@@ -45,11 +49,17 @@ const (
 func (builder conversionBuilder) propertyConversionHandler(
 	toProp *astmodel.PropertyDefinition,
 	fromType *astmodel.ObjectType,
-) []dst.Stmt {
+) ([]dst.Stmt, error) {
+	var err error
 	for _, conversionHandler := range builder.propertyConversionHandlers {
-		stmts, matched := conversionHandler(toProp, fromType)
-		if matched || toProp.HasTagValue(ConversionTag, NoARMConversionValue) {
-			return stmts
+		conversion, convErr := conversionHandler(toProp, fromType)
```

Comment:
is `convErr` actually needed here? I don't think it is?

Just repeatedly assign to `err`, check if it's not nil and break if one was found?

Reply:
Linter complained about `err` shadowing. 

I can reuse the existing `err` only if I create a separate declaration for `conversion` and then switch from `:=` to `=`, and using a new variable name seemed simpler.

Reply:
I've reworked it to avoid the odd name for an error.